src/gax-internal/tests/grpc_timeout.rs

```diff
@@ -0,0 +1,243 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[cfg(all(test, feature = "_internal_grpc_client"))]
+mod test {
+    use auth::credentials::testing::test_credentials;
+    use gax::options::*;
+    use gax::retry_policy::{AlwaysRetry, RetryPolicyExt};
+    use gax::retry_throttler::{CircuitBreaker, RetryThrottlerArg};
+    use google_cloud_gax_internal::grpc;
+    use grpc_server::{builder, google, start_echo_server};
+    use std::sync::{Arc, Mutex};
+    use std::time::Duration;
+
+    type Result<T> = std::result::Result<T, Box<dyn std::error::Error>>;
```

Comment:
I started using `anyhow::Result<>` in new tests, less typing.

Reply:
Ack, I will follow suit.